src/client/store/ui/userManagement/actions/updateUser.ts

```diff
@@ -3,25 +3,24 @@ import { Functions } from '@utils/functions'
 import axios from 'axios'
 
 import { ApiEndPoint } from '@meta/api/endpoint'
-import { CycleParams } from '@meta/api/request'
+import { CountryIso } from '@meta/area'
 import { User } from '@meta/user'
 
-type Params = CycleParams & {
+type Params = {
+  countryIso: CountryIso
   user: User
   profilePicture: File | null
 }
 
 const putUser = Functions.debounce(
   async (props: Params) => {
-    const { user, profilePicture, countryIso, assessmentName, cycleName } = props
+    const { user, profilePicture, countryIso } = props
 
     try {
       const formData = new FormData()
       formData.append('profilePicture', profilePicture)
       formData.append('user', JSON.stringify(user))
       formData.append('countryIso', countryIso)
```

Comment:
have you tested your auth middleware still works in a national correspondent edits a collaborator ?

Reply:
it should, but I make another test

```
express.put(ApiEndPoint.User.many(), multer().single('profilePicture'), AuthMiddleware.requireEditUser, updateUser)
```

```
const requireEditUser = async (req: Request, _res: Response, next: NextFunction) => {
  const { countryIso } = <Record<string, string>>{ ...req.params, ...req.query, ...req.body }
  const user = Requests.getRequestUser(req)
  _next(Users.getRolesAllowedToEdit({ user, countryIso: countryIso as CountryIso }).length > 0, next)
}
```

```
if (isNationalCorrespondent(user, countryIso) || isAlternateNationalCorrespondent(user, countryIso)) {
  return [RoleName.COLLABORATOR, RoleName.VIEWER]
}
```